src/Simulation/Simulators/SparseSimulator/README.md

```diff
@@ -0,0 +1,155 @@
+# Introduction
+This is a an alternative quantum simulator, compatible with Q\# and Microsoft's quantum development kit. It is efficient for algorithms with a smaller number of states in superposition, regardless of the number of qubits.
+
+# Repository Structure
+
+/SparseQuantumSimulator/
+|--/Native/: C++ code for the simulator
+|--/SparseSimulatorCS/: C# library for the simulator 
+|--/SparseSimQSharpTests/: Several Q# tests of the simulator
+|--/SparseSimulatorTests/: C++ tests of the simulator
```

Comment:
You may want to add double space `  ` in the end of every line. Otherwise currently in the rendered version all the lines are merged into a single line. 

You can see the rendered version if you press "Display the reach diff" link in the top-right corner of this file's diff (third from the right, after `...` and "Viewed").

Reply:
Addressed it here: https://github.com/microsoft/qsharp-runtime/pull/769

Reply:
Addressed in https://github.com/sam-jaques/qsharp-runtime/pull/1